resources/scss/ceres/views/Templates/BasketItemList/_basket-list-item.scss

```diff
@@ -244,62 +244,93 @@
 }
 
 /* Compatibility for IE */
-html.ie .checkout .checkout-rightside  {
+html.ie #vue-app .basket-item-container {
```

Comment:
Wird der Verweis auf `#vue-app` hier benötigt? Selekoren mit ID sollten vermieden werden.

Reply:
fixed and tested